unittests/AST/ASTImporterTest.cpp

```diff
@@ -300,6 +306,9 @@ class ASTImporterTestBase : public ::testing::TestWithParam<ArgVector> {
                &ToCtx = ToAST->getASTContext();
     ASTImporter Importer(ToCtx, ToAST->getFileManager(), FromCtx,
                          FromCtx.getSourceManager().getFileManager(), false);
+    It->Unit->beginSourceFile();
```

Comment:
`TestBase::Import` may be called several times. Is it ok to call `beginSourceFile()` twice on the same object?